File: HackerRank 30 Days/day2/main.js
function solve(mealCost, tipPercent, taxPercent) {
    const tip = mealCost * (tipPercent / 100);
    const tax = mealCost * (taxPercent / 100);
    const totalCost = mealCost + tip + tax;

    console.log(Math.round(totalCost));
}

const inputLines = require("fs").readFileSync(0, "utf8").trim().split(/\r?\n/);
const mealCost = parseFloat(inputLines[0]);
const tipPercent = parseInt(inputLines[1], 10);
const taxPercent = parseInt(inputLines[2], 10);

solve(mealCost, tipPercent, taxPercent);

File: HackerRank 30 Days/day2/main.py
import math

meal_cost = float(input().strip())
tip_percent = int(input().strip())
tax_percent = int(input().strip())

tip = meal_cost * tip_percent / 100
tax = meal_cost * tax_percent / 100
total_cost = meal_cost + tip + tax

print(int(round(total_cost)))
